### PR TITLE
CB-2681-CB-2682 Configure Providers for Schema Registry and SMM

### PR DESCRIPTION
**Pull request by smaniraju**
Configure Providers for Schema Registry Server and Streams Messaging Manager.
1. Configure RDS Providers for Schema Registry Server and SMM.
2. Configure Local CM Admin and Password for SMM

Closes #CB-2681
Closes #CB-2682

### CODE REVIEW

File: template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.AbstractRdsRoleConfigProvider;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.template.views.RdsView;
+
+@Component
+public class SchemaRegistryServiceConfigProvider extends AbstractRdsRoleConfigProvider {
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java]: Not a big deal but the class name is a little bit misleading, as no service-level configs are affected here (only DB-related role configs).

Comment (smaniraju) [reply template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java]: File Renamed.

**Comment by doktoric** [comment]
@smaniraju can you please add the necessary changes to SupportedDatabaseProvider.java ?

File: template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java (hunk 1 above)
Comment (lajosrodek) [reply template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java]: 👍

**Pull request merged**